redirect/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "create" {
+  description = "Should resources be created"
+  default     = true
+  type        = bool
+}
+
+variable "project" {
+  description = "Kebab-cased name of the project. Will be included in resource names"
+  type        = string
+}
+
+variable "environment" {
+  description = "Kebab-cased name of the environment, eg. production, staging, development, preview. Will be included in resource names"
+  type        = string
+}
+
+variable "name_prefix" {
```

Comment:
seems to only be used for the bucket name and there's a separate `bucket` variable for overriding the bucket name, so maybe `name_prefix` is unnecessary?

Reply:
👍